sys/shell/commands/sc_gnrc_netif.c

```diff
@@ -1175,6 +1192,9 @@ static int _netif_set(char *cmd_name, netif_t *iface, char *key, char *value)
     else if ((strcmp("channel", key) == 0) || (strcmp("chan", key) == 0)) {
         return _netif_set_u16(iface, NETOPT_CHANNEL, 0, value);
     }
+    else if ((strcmp("checksum", key) == 0)) {
+        return _netif_set_bool(iface, NETOPT_CHECKSUM, value);
+    }
```

Comment:
This is already implemented. See line 75

Reply:
Oh, I see..